### PR TITLE
v1.0 Release Candidate

### PR DESCRIPTION
**Pull request by chris-cheshire**

### CODE REVIEW

**Comment by maxulysse** [comment]
I think all test related data should be in the `nf-core/test-datasets` repo

**Review by maxulysse — DISMISSED** [review]
Really like what you did, I think there's a couple of things to update, but we should be pretty close to a release

File: .nf-core.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+files_unchanged:
+  - .github/workflows/linting.yml
+  - assets/multiqc_config.yaml
+  - assets/sendmail_template.txt
+  - lib/NfcoreSchema.groovy
+  - docs/README.md
+  - bin/markdown_to_html.py
+files_exist:
+  - .github/markdownlint.yml
```
[the remark this answers is not in the document]
Comment (maxulysse) [reply .nf-core.yaml:9]: Did you try removing that?

File: assets/multiqc_config.yaml [hunk 2]
```diff
@@ -9,3 +9,109 @@ report_section_order:
...
+#   - "_processedFile"
+# extra_fn_clean_trim:
+#   - "#"
+#   - ".myext"
+
+# Customise the module search patterns to speed up execution time
+#  - Skip module sub-tools that we are not interested in
+#  - Replace file-content searching with filename pattern searching
+#  - Don't add anything that is the same as the MultiQC default
+# See https://multiqc.info/docs/#optimise-file-search-patterns for details
```
Comment (maxulysse) [inline assets/multiqc_config.yaml:74]: Can remove comments

File: conf/laptop.config [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/*
```
[the remark this answers is not in the document]
Comment (maxulysse) [reply conf/laptop.config:1]: I'm still not sure we should have such a configuration here actually

File: modules/local/get_chrom_sizes.nf [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    path '*.sizes'      , emit: sizes
+    path '*.fai'        , emit: fai
+    path "*.version.txt", emit: version
+
+    script:
+    def software = 'samtools'
+    """
+    samtools faidx $fasta
+    cut -f 1,2 ${fasta}.fai > ${fasta}.sizes
+    echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//' > ${software}.version.txt
```
Comment (maxulysse) [inline modules/local/get_chrom_sizes.nf:37]: I think you can make it as a nf-core/modules, it's an easy command, but could be worth it to share with others

File: .gitignore [hunk 5]
```diff
@@ -3,7 +3,22 @@ work/
...
+.local
+.config
+.bash_history
+
+R_sessionInfo.log
+
+dev/docker/static_reports/test_data/*
+dev/docker/static_reports/test_output/*
+!dev/docker/static_reports/test_data/_dummy.txt
+!dev/docker/static_reports/test_output/_dummy.txt
```
Comment (maxulysse) [inline .gitignore:19]: Why escaping this file if not in the repo?

**Review by maxulysse — COMMENTED** [review]
There are a couple of comments left by the reviewers that are not yet addressed.
Apart from that, it's looking super good.

File: assets/multiqc_config.yaml (hunk 2 above)
Comment (chris-cheshire) [reply assets/multiqc_config.yaml:74]: Done

File: modules/local/get_chrom_sizes.nf (hunk 4 above)
Comment (chris-cheshire) [reply modules/local/get_chrom_sizes.nf:37]: Ok I will get this submitted

File: .gitignore (hunk 5 above)
Comment (chris-cheshire) [reply .gitignore:19]: Yep, I have cleaned up this file now

**Comment by chris-cheshire** [comment]
@maxulysse I cant see the unaddressed comments? thanks again for your review - I have resolved your comments. I will submit the get_chrom_sizes module, but if poss I would like to press ahead with release rather than waiting for that to be accepted into the modules repo.

**Review by drpatelh — APPROVED** [review]
Great work @chris-cheshire ! 😍

File: conf/laptop.config (hunk 3 above)
[the remark this answers is not in the document]
Comment (mashehu) [reply conf/laptop.config:1]: I agree, you should untrack this file.

File: docs/output.md [hunk 6]
```diff
@@ -1,35 +1,71 @@
...
+        - [SEACR peak calling](#seacr-peak-calling)
+        - [BEDtools](#bedtools)
+    - [Reporting](#reporting)
+        - [Python reporting](#python-reporting)
+        - [MultiQC](#multiqc)
+        - [IGV](#igv)
+        - [Deeptools](#deeptools)
+    - [Workflow reporting and genomes](#workflow-reporting-and-genomes)
+        - [Reference genome files](#reference-genome-files)
+        - [Pipeline information](#pipeline-information)
```
Comment (mashehu) [inline docs/output.md:39]: No need for a markdown based ToC anymore. It will be generated from the headings on the website automatically

**Pull request merged**